### ISSUE TITLE
Name collision with PyPI parcel

### ISSUE THREAD

**Issue by jstaf** [LabAdvComp/parcel#83]
Ran into an issue today where your parcel package has the same name as this one from PyPI: https://pypi.python.org/pypi/parcel/

**Comment by serverhorror** [comment]
I have a similiar problem. NCI-GDC/gdc-client uses your package as a dependency and it seems problematic to be able ever pip install the gdc-client as long as this naming conflict exists.

**Comment by jstaf** [comment]
Yes, that was exactly how I ran into this issue. pip installation of gdc-client gets really weird because of this package's name.

**Comment by bmilash** [comment]
Yup, just ran into this myself installing gdc-client.
